src/assets/specs/command-api.yaml

```diff
@@ -154,11 +154,11 @@ paths:
           description: The category is not recognized.
 
 
-  /v0/diagnostic_check:
+  /v0/diagnostic_checks:
     get:
       operationId: diagnosticForCategoryAndID
       summary: >-
-        Return the check for the given category and checkID, or 404.
+        Return all the checks, optionally filtered by specified category and/or checkID, or 404.
```

Comment:
The response no longer lists 404 as an option.

Reply:
It never 404s, and again this wasn't updated because nothing is checking it and I forgot to check it as well.